howto/anbox-runtime/develop-addon-devmode.md

```diff
@@ -10,12 +10,12 @@ This guide explains how to use an instance in development mode to develop and te
 Start a raw instance with `--devmode` enabled:
 
 ```
-amc launch --devmode --raw
+amc launch --devmode
```

Comment:
As discussed in SU, dropping `--raw` here would cause the instance to fail to launch.
```
$ amc launch --devmode
Error: accepts 1 arg(s), received 0
```
This will require code changes to the ams client if we suggest people to the above command.

Same below

Reply:
https://github.com/canonical/ams/pull/996
Simon has submitted a PR and made a change on amc to support the above case.